aptos-move/framework/aptos-framework/sources/coin.move

```diff
@@ -256,6 +257,14 @@ module aptos_framework::coin {
         option::none()
     }
 
+    public fun apt_fa_metadata(): Object<Metadata> {
+        // if object::exists(@aptos_fungible_asset) {
+        //     return object::address_to_object<Metadata>(@aptos_fungible_asset)
+        // } else {
+        // }
+        object::address_to_object<Metadata>(@aptos_fungible_asset)
```

Comment:
This call has a significant perf cost for no-op txn. Could you add a friend function in `object.move` called `address_to_object_unchecked` that checks NEITHER `ObjectCore` nor `T`(metadata here)